server/src/Command.cpp

```diff
@@ -99,6 +99,7 @@ std::ostream &operator<<(std::ostream &os, const Command::Action &action) {
 std::ostream &operator<<(std::ostream &os, const Command &command) {
   os << "Command{action=" << command.action << ", "
      << "file='" << command.file << "', "
+     << "dir='" << command.dir << "', "
```

Comment:
The only use we have for now is the build-dir? I prefer the use of `buildDirectory` for now.